src/app/search/directives/series-list.component.css

```diff
@@ -0,0 +1,43 @@
+publish-series-card, .series {
+  display: block;
+  overflow: hidden;
+  position: relative;
+  margin-top: 20px;
+  margin-right: 20px;
+  flex-basis: 490px;
+  -ms-flex-preferred-size: 490px;
+  -webkit-flex-basis: 490px;
+  -webkit-box-flex: 2;
+  -moz-box-flex: 2;
+  -o-box-flex: 2;
+  -ms-box-flex: 2;
+  box-flex: 2;
+  -webkit-flex-grow: 2;
+  flex-grow: 2;
+  -webkit-flex-shrink: 1;
+  flex-shrink: 1;
+  height: 192px;
+  background: #fff;
+}
+
+/* attempt to fix the odd last item stretching
```

Comment:
I wanted adaptive cards, I got adaptive cards. I don't yet know what to do about the last odd card that grows to the size of the row. It is not a good look.